sdk/test/unit/bitmap.test.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  bitmapToBoolArray,
+  flipBit,
+  getBit,
+  boolArrayToBitmap,
+} from '../../src/bitmap';
+import {getEmpty256Array} from '../utils';
+import {BigNumber} from '@ethersproject/bignumber';
+
+describe('bitmap', () => {
+  describe('flipBit', () => {
```

Comment:
Here you should explicitly test for the wrap-around so that changes in the future preserve that behaviour